crates/bevy_render/src/lib.rs

```diff
@@ -190,8 +190,8 @@ pub mod main_graph {
 }
 
 #[derive(Resource)]
-struct FutureRendererResources(
-    Arc<
+pub struct FutureRendererResources(
```

Comment:
Struct needs documentation.

Reply:
I have added documentation. If this documentation is not good enough, or you have better doc comments in mind, please let me know.